src/main/java/core/basesyntax/factory/AbstractFactory.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.factory;
+
+import core.basesyntax.controller.BetConsoleHandler;
+import core.basesyntax.controller.UserConsoleHandler;
+import core.basesyntax.exception.NoSuchInjectionObjectException;
+import core.basesyntax.lib.Dao;
+import java.util.Map;
+
+public class AbstractFactory {
```

Comment:
I think this class is redundant

we already have a Factory that provides us the result we need

Reply:
done